app/code/core/Mage/Adminhtml/controllers/Catalog/Product/ReviewController.php

```diff
@@ -97,7 +97,7 @@ public function editAction()
         $this->_title($this->__('Edit Review'));
 
         $this->loadLayout();
-        $this->_setActiveMenu('catalog/review');
+        $this->_setActiveMenu('catalog/reviews_ratings/reviews/all');
```

Comment:
Edit can come from either _Pending Reviews_ or _All Reviews_:
```suggestion
        if ($this->getRequest()->getParam('ret') === 'pending') {
            $this->_setActiveMenu('catalog/reviews_ratings/reviews/pending');
        } else {
            $this->_setActiveMenu('catalog/reviews_ratings/reviews/all');
        }

Reply:
Yes!